views/courseEdit.ejs

```diff
@@ -0,0 +1,65 @@
+<%- contentFor('content') %>
+<div class="container col-md-4 col-md-offset-4">
+  <h1 class="text-center">Update Semester</h1>
+  <form id="course_form" action="/courses/<%=course.semester_name%>/<%=course.module_code%>" role="form" method="post">
```

Comment:
```suggestion
  <form id="course_form" action="/courses/<%=encodeURIComponent(course.semester_name)%>/<%=encodeURIComponent(course.module_code)%>" role="form" method="post">
```

Use encodeURIComponent(), then there is no need to use * in the routes. Refer to #42